azure/v12.0.0/README.md

```diff
@@ -2,7 +2,12 @@
 
 This is the first release to support kubernetes 1.17 on Azure.
 
-Apart from this important change, there have been a high number of changes under the hood, including a few important
+As of this release NGINX Ingress Controller App is optional and not pre-installed component on Azure.
+Upgrading existing clusters with pre-installed NGINX will leave NGINX unchanged.
+This allows NGINX App installations to be managed independently from the base platform lifecycle.
```

Comment:
@cokiengchiara @MartaD can you help how to wordsmith it - that's it not only privilege but also extra duty to keep NGINX App up-to-date, since app updates (not even patch releases) are not automatic, at least not yet.